exercises/practice/fizz-buzz/.meta/config.json

```
@@ -0,0 +1,12 @@
+{
+  "blurb": "Return 'Fizz', 'Buzz' or 'FizzBuzz' for numbers divisible by 3 or 5 but not both.",
+  "authors": ["marianfoo"],
+  "contributors": [
+  ],
+  "files": {
+    "solution": ["zcl_fizz_buzz.clas.clas.abap"],
+    "test": ["zcl_fizz_buzz.clas.clas.testclasses.abap"],
+    "example": [".meta/zcl_fizz_buzz.clas.clas.abap"]
```

looks like there's one `clas` to many in each here:
```suggestion
    "solution": ["zcl_fizz_buzz.clas.abap"],
    "test": ["zcl_fizz_buzz.clas.testclasses.abap"],
    "example": [".meta/zcl_fizz_buzz.clas.abap"]
```